src/h5web/App.tsx

```diff
@@ -60,7 +50,7 @@ function App(): JSX.Element {
               entity={selectedEntity}
             />
           ) : (
-            <DatasetVisualizer dataset={selectedDataset} />
+            <Visualizer entity={selectedEntity} />
```

Comment:
I introduce a new top-level component called `Visualizer` that decides which "visualizer" to show, if any.